src/main/java/eu/europeana/metis/sandbox/entity/RecordLogEntity.java

```diff
@@ -24,13 +24,14 @@
   @Enumerated(EnumType.STRING)
   private Status result;
 
-  @Column
+  @Column(columnDefinition = "VARCHAR(MAX)")
```

Comment:
I think this column definition is very implementation dependent (i.e. not all databases support this or interpret it the same way). 
You should probably choose a sensible limit and truncate the error message here so that if it is bigger than your limit you don't create an error (which it currently does).

Note: this declaration also occurs above where the content is stored. I think this is often bigger than whatever 'MAX' is, and should probably be stored using a TEXT or even BLOB style column.

Reply:
columnDefinition affects DDL generation, I have it for the in memory database when running the integration tests. But persistent postgres is not generating the database from this definition, the table itself must exist or the application will fail, and the field error in the DB can be something else like TEXT or other type. But I see it can be confusing, I move the definition of this type to a custom script for the integration test